modules/json/src/helpers/convert-functions.js

```diff
@@ -28,7 +28,7 @@ export default function convertFunctions(props, configuration) {
     }
 
     // Invalid functions return null, show default value instead.
-    if (propValue) {
+    if (propValue !== null) {
```

Comment:
Non-accessor props are allowed to be null. You must treat the return values of functions as its own special case.